src/KeyworkerDashboard/KeyworkerDashboard.js

```diff
@@ -27,23 +29,27 @@ class KeyworkerDashboard extends Component {
   }
 
   async loadStatsForPeriod(duration, period) {
-    const { agencyId, handleError, dispatchLoaded, dispatchStats } = this.props
-    dispatchLoaded(false)
-    try {
-      const format = 'YYYY-MM-DD'
-      const fromDate = moment()
-        .subtract(duration, period)
-        .format(format)
-      const toDate = moment().format(format)
+    const { agencyId, handleError, dispatchLoaded, dispatchStats, migrated, history } = this.props
+    if (!migrated) {
```

Comment:
don't like this but is needed for the special case of switching caseload and remaining on the same page